src/consumer/component/dependencies-resolver.js

```diff
@@ -325,6 +326,19 @@ Try to run "bit import ${componentId} --objects" to get the component saved in t
   return { componentsDeps, devComponentsDeps, packagesDeps, devPackagesDeps, untrackedDeps, relativeDeps, missingDeps };
 }
 
+function findPeerDependencies(consumerPath: string): Object {
```

Comment:
this logic should be in bit-js in the build-tree file.
There is also some logic related to peerDeps (i'm not sure that alone will be enough for you), but anyway, the correct place is there for sure.

Reply:
I agree. It just takes too much time to work with bit-javascript. I'll add a todo comment to move it there once we have the mono repo.